app/views/chapters/show.html.erb

```diff
@@ -13,20 +13,25 @@
       <p class="subtitle">- don't be shy -</p>
       <%= link_to "{ Join Us }", "http://www.meetup.com/#{@chapter.meetup}", :class=>"button" %>
 
-      <!-- <% if @chapter.sponsors.count != 0 %>
+      <% if @chapter.sponsors.count != 0 %>
         <div class="hide">
           <div class="sponsors">
             <h2>Sponsors</h2>
-            <div id="logos">
+            <div class="logos">
               <% @sponsors.each do |sponsor| %>
                 <% if sponsor.image? %>
```

Comment:
We should probably add an `<% else %>` statement in here for instances where there is no sponsor logo.